README.md

```diff
@@ -1,59 +1,162 @@
+To learn more about connecting to your Capella cluster, please follow the [instructions](https://docs.couchbase.com/cloud/get-started/connect.html).
 
-### Dependency Injection via DBSetupRunner class 
+Specifically, you need to do the following:
 
-The quickstart code provides a CommandLineRunner called DBSetupRunner in the runners folder that wires up the Bucket and Cluster objects for dependency injection.  This runner also creates the bucket, collection, scope, and indexes for the tutorial to run properly automatically when the application 
+- Create the [database credentials](https://docs.couchbase.com/cloud/clusters/manage-database-users.html) to access the travel-sample bucket (Read and Write) used in the application.
+- [Allow access](https://docs.couchbase.com/cloud/clusters/allow-ip-address.html) to the Cluster from the IP on which the application is running.
+
+All configuration for communication with the database is read from the environment variables. We have provided a convenience feature in this quickstart to read the environment variables from a local file, `application.properties` in the `src/main/resources` folder.
+
+```properties
+spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
+spring.couchbase.bucket.name=travel-sample
+spring.couchbase.bootstrap-hosts=DB_CONN_STR
+spring.couchbase.bucket.user=DB_USERNAME
+spring.couchbase.bucket.password=DB_PASSWORD
+```
+
+Instead of the hash symbols, you need to add the values for the Couchbase connection.
+
+> Note: The connection string expects the `couchbases://` or `couchbase://` part.
 
 ## Running The Application
 
-At this point the application is ready and you can run it via your IDE or from the terminal:
+### Directly on Machine
+
+At this point, we have installed the dependencies, loaded the travel-sample data and configured the application with the credentials. The application is now ready and you can run it.
 
 ```sh
-mvn spring-boot:run -e -X
+mvn spring-boot:run
 ```
 
-You can launch your browser and go to the [Swagger start page](https://localhost:8080/swagger-ui/index.html).
+### Using Docker
 
-## Running The Tests
+Build the Docker image
 
-To run the standard integration tests (which requires a running Couchbase Server), use the following command:
+```sh
+docker build -t java-springboot-quickstart .
+```
+
+Run the Docker image
 
 ```sh
-mvn verify
+docker run -d --name springboot-container -p 8080:8080 java-springboot-quickstart
 ```
 
-## Project Setup Notes
+Note: The `application.properties` file has the connection information to connect to your Capella cluster. These will be part of the environment variables in the Docker container.
```

Comment:
Is this correct? Your docker container already has the credentials hardcoded into it via the application.properties right?

Reply:
right

Reply:
Have replaced this with conditional fetching from environment variables instead of hardcoding